packages/glimmer-compiler/lib/compiler.ts

```diff
@@ -19,7 +20,14 @@ export type TemplateSpec = string;
  * @return {TemplateSpec} A template spec string
  */
 export function compileSpec(string: string, options: any): TemplateSpec {
+  let meta: BlockMeta = {};
+  if (typeof options === 'object' && options.meta.moduleName && options.meta.moduleName) {
```

Comment:
it should treat meta as opaque, also, I think options is just meta, cc @rwjblue ?